### PR TITLE
Add example text to external file

### PR DESCRIPTION
**Pull request by techytushar**
Signed-off-by: Tushar Mittal <chiragmittal.mittal@gmail.com>

### CODE REVIEW

**Comment by pombredanne** [comment]
Thanks! You are on a roll!

**Comment by pombredanne** [comment]
Why is it better to have this in an external file?

**Comment by techytushar** [comment]
there was a comment in the code saying `FIXME: this should be pushed out in some external help or pushed down in plugins.`, besides IMHO this makes the `cli.py` file clean...

File: src/scancode/cli.py [hunk 1]
```diff
@@ -82,6 +82,7 @@
 from scancode.utils import BaseCommand
 from scancode.utils import path_progress_message
 from scancode.utils import progressmanager
+from scancode.example_text import examples_text, epilog_text
```
Comment (pombredanne) [inline src/scancode/cli.py]: Make each import on its own line

**Comment by pombredanne** [comment]
I can buy that, but the file should be called help.py

**Comment by pombredanne** [comment]
Thanks! If we try to go a tad further, what would be the way to let plugins provide their own sample or extended help instead?

**Comment by pombredanne** [comment]
Thanks. Merging!

**Pull request merged**